### PR TITLE
Feature/Add new information groups for RiskResponse to include missing fields

### PR DESCRIPTION
**Pull request by asilvabe**
In this update, new methods were added to the `Messages\RiskResponse` class that group fields from the EmailAge response that were not being returned.

### Changed
-  In the `emailInformation()` method, the following data was added: `company`, `email_creation_days`, `domain_creation_days`, `fraudRisk`
- In the `ipInformation()` method, the `ip_risklevel` and `ip_risklevelid` fields were removed, which were moved to the new `ipRiskInformation()` method.

### Added
- Added a changelog.md file to track project updates.
- Added the `riskBandMessage()` method which returns the `EAReason` field.
- Added the method `disOverallScore()` which returns the `overallDigitalIdentityScore` field corresponding to the Digital Identity Score (DIS) product information.
- Added the method `disOverallScoreDescrition()` which returns the `disDescription` field corresponding to the Digital Identity Score (DIS) product information.
- Added the method `addressInformation()` which groups information about customer's shipping or billing address related fields.
- Added the method `digitalIdentityScoreInformation()` which groups information about Digital Identity Score related fields.
- Added the method `ipRiskInformation()` which groups information about the IP and device risk related fields.
- Added the method `ipLocationInformation()` which groups information about the IP location related fields.
- Added the method `ownerInformation()` which groups information about the email owner related fields.
- Added the method `phoneInformation()` which groups information about the phone related fields.
- Added the method `socialMediaInformation()` which groups information about the email social media related fields.

### Fixed
- Fixed getting the `hits` field in the `emailInformation()` method

### CODE REVIEW

**Review by luisfelipegh — APPROVED** [review]

**Review by luisfelipegh — APPROVED** [review]

File: src/Messages/RiskResponse.php [hunk 1]
```diff
@@ -196,30 +259,122 @@ public function emailInformation()
...
+     * @return array
+     */
+    public function ipRiskInformation()
+    {
+        if (empty($this->resultData('ipaddress'))) {
+            return [];
+        }
+
+        return [
+            'level' => $this->resultData('ip_risklevelid'),
```
Comment (luisfelipegh) [inline src/Messages/RiskResponse.php:299]: Este no es el mismo de la linea 282 ?
?

File: src/Messages/RiskResponse.php [hunk 2]
```diff
@@ -196,30 +259,122 @@ public function emailInformation()
...
+            'levelMessage' => $this->resultData('ip_risklevel'),
```
Comment (luisfelipegh) [inline src/Messages/RiskResponse.php:300]: Este no es el mismo de la linea 283 ?

File: src/Messages/RiskResponse.php [hunk 3]
```diff
@@ -157,6 +167,53 @@ public function fraudType()
...
+
+    /**
+     * Groups information about Digital Identity Score related fields.
+     * @return array
+     */
+    public function digitalIdentityScoreInformation()
+    {
+        return [
+            'description' => $this->resultData('disDescription'),
+            'overallScore' => $this->resultData('overallDigitalIdentityScore'),
```
Comment (luisfelipegh) [inline src/Messages/RiskResponse.php:193]: Agreguemos tambien un metodo para obtener este overallScore independiente mente como el EAScore

File: src/Messages/RiskResponse.php [hunk 4]
```diff
@@ -157,6 +167,53 @@ public function fraudType()
...
+    }
+
+    /**
+     * Groups information about Digital Identity Score related fields.
+     * @return array
+     */
+    public function digitalIdentityScoreInformation()
+    {
+        return [
+            'description' => $this->resultData('disDescription'),
```
Comment (luisfelipegh) [inline src/Messages/RiskResponse.php:192]: Agreguemos tambien un metodo para obtener este overallScore description independientemente como el EAReason

**Review by luisfelipegh — COMMENTED** [review]
Revisar los comentarios

File: src/Messages/RiskResponse.php (hunk 1 above)
Comment (asilvabe) [reply src/Messages/RiskResponse.php:299]: Corregido. Removí el campo del método `ipInformation()` y lo dejé únicamente en `ipRiskInformation()`

File: src/Messages/RiskResponse.php (hunk 2 above)
Comment (asilvabe) [reply src/Messages/RiskResponse.php:300]: Corregido. Removí el campo del método `ipInformation()` y lo dejé únicamente en `ipRiskInformation()`

File: src/Messages/RiskResponse.php (hunk 3 above)
Comment (asilvabe) [reply src/Messages/RiskResponse.php:211]: Método agregado

File: src/Messages/RiskResponse.php (hunk 4 above)
Comment (asilvabe) [reply src/Messages/RiskResponse.php:210]: Método agregado

**Review by luisfelipegh — APPROVED** [review]

**Pull request merged**